test/new-e2e/tests/agent-subcommands/flare/flare_win_test.go

```diff
@@ -9,24 +9,24 @@ package flare
 import (
 	"testing"
 
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client/agentclient"
+	"github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+
 	"github.com/stretchr/testify/assert"
 )
 
-type windowsFlareSuite struct {
-	baseFlareSuite
-}
+type windowsFlareSuite baseFlareSuite
```

Comment:
I'm not sure this is strictly equivalent (eg. for casting one type to another), any reason for this change ?

Reply:
It's not equivalent indeed (not only in terms of cast. Both works in this case, but still not the same).
We should favour embed, this is a remain from some testing with type declaration.